db/db_filesnapshot.cc

```diff
@@ -53,7 +53,7 @@ Status DBImpl::GetLiveFiles(std::vector<std::string>& ret,
   std::vector<uint64_t> live_table_files;
   std::vector<uint64_t> live_blob_files;
   for (auto cfd : *versions_->GetColumnFamilySet()) {
-    if (cfd->IsDropped()) {
+    if (cfd->IsDropped() || cfd->ioptions().is_transient) {
```

Comment:
Users may want to get SSTs from the transient CF with GetLiveFiles() or GetLiveFilesStorageInfo(), we can make this optional and have checkpoint and backup to specify to skip files from the transient CF.